pkg/reconciler/instances/istio/actions/performer.go

```diff
@@ -201,7 +202,7 @@ func (c *DefaultIstioPerformer) PatchMutatingWebhook(context context.Context, ku
 		if err != nil {
 			return err
 		}
-		err = c.addLabelSelectorToWebhook(whConf, webhookNameToChange, requiredLabelSelector)
+		err = c.setLabelSelectorToWebhook(whConf, webhookNameToChange, requiredLabelSelector)
```

Comment:
Why did you change the function signature? I don't necessarily find `set` word in this context as appropriate. In OOP languages it would imply that there could be a hard assignment performed underneath which is not the case here. We in fact include one more label selector the list of matched expressions if it not present there. I know what you wanted to achieve - to stay idempotent we should not add it twice, so add would imply we mindlessly include it to the list. How about renaming it to: `addLabelSelectorfNotPresent`? Or maybe even: `addLabelSelectorfNotPresentin`? Reading it out loud with parameters names should suggest that we should validate the existence in webhooks so we don't need to include it in function name.

Reply:
I've changed it to `addNamespaceSelectorIfNotPresent`